src/core_plugins/kibana/public/dashboard/panel/panel_header/panel_header_container.js

```diff
@@ -24,13 +24,18 @@ import {
 
 const mapStateToProps = ({ dashboard }, { panelId }) => {
   const embeddable = getEmbeddable(dashboard, panelId);
+  console.log("embeddable", embeddable);
   const panel = getPanel(dashboard, panelId);
```

Comment:
Looks like you left some stray console logs in here. Probably should remove these.